support/AndroidManifest.xml

```diff
@@ -1 +1 @@
-<manifest package="fmt" />
\ No newline at end of file
+<manifest package="org.fmtlib" />
```

Comment:
This should probably be `net.fmtlib` because that's the fmtlib's domain (reversed).